packages/salesforcedx-vscode-apex/test/vscode-integration/settings.test.ts

```diff
@@ -28,15 +28,15 @@ describe('Settings', () => {
 
   describe('useApexLibrary', () => {
     it('should return true if configuration value is true', () => {
-      settingStub.withArgs('experimental.useApexLibrary').returns(true);
+      // settingStub.withArgs('experimental.useApexLibrary').returns(true);
 
-      expect(useApexLibrary()).to.equal(true);
+      // expect(useApexLibrary()).to.equal(true);
     });
 
     it('should return false if configuration value is false', () => {
-      settingStub.withArgs('experimental.useApexLibrary').returns(false);
+      // settingStub.withArgs('experimental.useApexLibrary').returns(false);
 
-      expect(useApexLibrary()).to.equal(false);
+      // expect(useApexLibrary()).to.equal(false);
     });
   });
```

Comment:
I think this suite can be removed right?

Reply:
oh whoops, forgot to clean this up